### ISSUE TITLE
feature request: "managed_charging_start_time"

### ISSUE THREAD

**Issue by Enno-von-Schwerin** [hombach/ioBroker.teslafi#29]
please add value "managed_charging_start_time" to the objects read from teslafi_API

to see if the car will start charging immedeatly after pluging in or at a specified time.

**Comment by Enno-von-Schwerin** [comment]
and add the value: "managed_charging_start_time" as well to see witch time the car will start charging.

**Comment by hombach** [comment]
guess the first value you wanna need is "managed_charging_active"  - two times the same is probably a typo.

**Comment by Enno-von-Schwerin** [comment]
if charging is stopped,
charger is plugged in
and time is set for charging start,

JSON looks like:
![image](https://github.com/user-attachments/assets/a8a196d0-000e-48d5-9d50-3cf449ad5f1e)

see marked lines

**Comment by hombach** [comment]
time code to be converted in lokal time:

```
import { fromUnixTime, format } from 'date-fns';

function convertUnixToLocalTime(unixTimestamp: number, dateFormat = "dd.MM.yyyy HH:mm:ss"): string {
    const date = fromUnixTime(unixTimestamp);
    return format(date, dateFormat);
}

// Example
const timestamp = 1731031200; // Unix-Timestamp
console.log(convertUnixToLocalTime(timestamp)); // z.B. "10.11.2024 00:00:00"

```

**Comment by hombach** [comment]
according to TeslaFi service the parameter "managed_charging_active" isn*t used.... so values are more or less "by accident".
So for this feature request parameter "scheduled_charging_start_time" will be used, only.

**Comment by hombach** [comment]
implemented for 0.4.0

**Comment by hombach** [comment]
0.4.0 released to NPM
